hardware/Schematic-Design-Guide.md

```diff
@@ -96,41 +125,59 @@ The matching effort strives to simultaneously achieve several goals:
   AN923 describes the matching method in detail.
 
    The matching circuits consist of impedance transformation block, Balun, and Low Pass Filter as following picture shows:
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-Topology.png)
-  The matching circuits are classified into 2 types due to the balun difference. One is for low frequency band(<500MHz), and another is for high frequency band(>500MHz)
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-LT-500MHz.png)
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-MT-500MHz.png)
+   <div align="center">
+    <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-Topology.png">  
+   </div>
+  
+  The matching circuits are classified into 2 types due to the balun difference. One is for low frequency band(<500MHz), and the other is for high frequency band(>500MHz)
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-LT-500MHz.png">  
+  </div>
+
+
+  <div align="center">
+  <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-MT-500MHz.png">  
+  </div>
 
   And more the matching circuits are different from components values which classified with output power levels and sub frequency bands:
 
   Matching components value table:
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Match-Circuits.png)
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\Sub-G-Match-Circuits.png">  
+  </div>
+
   Performance table:
-  ![](files/HW-Schematic-Design-Guide/Sub-G-Tested-Performance.png)
+  <div align="center">
+   <img src="files\HW-Schematic-Design-Guide\Sub-G-Tested-Performance.png">  
+  </div>
 
 - Matching circuits design for 2.4GHz band
   AN930 describes the 2.4GHz matching method in detail.The 2.4 GHz front end has a unified, single-ended TX and RX pin (2G4RF_IOP), so the TX and RX paths are tied together internally. The 2G4RF_ION TX pin has to be grounded at the pin. Externally, a single-ended matching network and harmonic filtering are required.
 
   2 main 2.4 GHz matching topologies are presented here:
 
   A ladder 2-element LC match for up to 10 dBm power levels:
```

Comment:
We should be clear this is for series 1.  It also makes sense to add Series 2 matches in this section?

Reply:
Added.